CMakeLists.txt

```diff
@@ -345,8 +356,31 @@ ELSE()
   ${CMAKE_BINARY_DIR}/aspect-gui
   @ONLY
   )
+INSTALL(FILES ${CMAKE_BINARY_DIR}/aspect-gui 
+	DESTINATION bin 
+	PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ
+		GROUP_EXECUTE GROUP_READ)
```

Comment:
Aren't executables usually installed in a way so that *everyone* can read and execute, not just user and group?